Bottom: 9c7a11a9132ef74f773a939f515e62f754c3f76a
Top:    e531696807585dd117f5cacf35f805f92bff759a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 18:59:19 +0000

Refresh of play-with-locked-object

---

diff --git a/src/draw.c b/src/draw.c
index faa9359..bd79e36 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -67,6 +67,7 @@
 #define	LARGE_TEXT_SIZE			3
 #define	N_TEXT_SIZES			4
 
+void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 
 /* ---------------------------------------------------------------------------
  * some local identifiers
